hack/cve-triage/cve_triage_core.py

```
@@ -0,0 +1,2288 @@
+        while cur and cur not in seen:
+            chain.append(cur)
+            seen.add(cur)
+            cur = edge_map.get(cur)
+        hops = len(chain) - 1
+        chain_lines = [f"  {chain[0]}"]
+        for step in chain[1:]:
+            chain_lines.append(f"    → {step}")
+        return [
+            f"✓ [{label}]: {node}  ({hops} hop{'s' if hops != 1 else ''})",
+            _cmd_str(node),
+        ] + chain_lines
```

Ruff (0.16.1)

[warning] Consider iterable unpacking instead of concatenation

Replace with iterable unpacking

(RUF005)